snap/hooks/configure

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+DEFAULT_DURATION=10
+
+duration="$(snapctl get duration)"
+
+if ! expr "$duration" : '^[0-9]*$' > /dev/null; then
+    echo "\"$duration\" is not a valid duration, reveting to $DEFAULT_DURATION seconds" >&2
+    snapctl set duration="$DEFAULT_DURATION"
+fi
```

Comment:
i guess you want to restart the service after changing the value here ... i.e. simply calling a restart at the end of the hook:

    snapctl restart "$SNAP_NAME".imv

beyond this the whole PR looks fine ...

Reply:
I added the restart after setting the duration setting. Thank you for the feedback!